### PR TITLE
Add product test for nested Hive views

### PR DESCRIPTION
**Pull request by alexjo2144**

### CODE REVIEW

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 1]
```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
                 queryAssert -> queryAssert.containsOnly(row("hive")));
     }
 
+    @Test
```
Comment (ebyhr) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:267]: nit: Specify `HIVE_VIEWS` group.
```suggestion
    @Test(groups = HIVE_VIEWS)
```

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 2]
```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
...
+    public void testNestedHiveViews()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_base_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_middle_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_top_view");
+
+        onHive().executeQuery("CREATE VIEW nested_base_view AS SELECT nationkey as k, name as n, regionkey as r, comment as c FROM nation");
```
Comment (ebyhr) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:274]: Prepend `n_` to each column name. This is CI failure cause. 
```suggestion
        onHive().executeQuery("CREATE VIEW nested_base_view AS SELECT n_nationkey AS k, n_name AS n, n_regionkey AS r, n_comment AS c FROM nation");
```

**Review by ebyhr — APPROVED** [review]

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 3]
```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
...
+        onHive().executeQuery("CREATE VIEW nested_middle_view AS SELECT n, c FROM nested_base_view WHERE k = 14");
+        onHive().executeQuery("CREATE VIEW nested_top_view AS SELECT n FROM nested_middle_view");
+
+        assertViewQuery(
+                "SELECT * FROM nested_top_view",
```
Comment (findepi) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:279]:
```suggestion
                "SELECT n_renamed FROM nested_top_view",
```

File: presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java [hunk 4]
```diff
@@ -264,6 +264,22 @@ public void testSimpleCoral()
...
+    @Test
+    public void testNestedHiveViews()
+    {
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_base_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_middle_view");
+        onHive().executeQuery("DROP VIEW IF EXISTS nested_top_view");
+
+        onHive().executeQuery("CREATE VIEW nested_base_view AS SELECT nationkey as k, name as n, regionkey as r, comment as c FROM nation");
+        onHive().executeQuery("CREATE VIEW nested_middle_view AS SELECT n, c FROM nested_base_view WHERE k = 14");
+        onHive().executeQuery("CREATE VIEW nested_top_view AS SELECT n FROM nested_middle_view");
```
Comment (findepi) [inline presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java:276]:
```suggestion
        onHive().executeQuery("CREATE VIEW nested_top_view AS SELECT n AS n_renamed FROM nested_middle_view");
```

**Comment by alexjo2144** [comment]
Updated, thanks

**Pull request merged**